DevTchap/Config/BuildSettings.swift

```diff
@@ -0,0 +1,423 @@
+// 
+// Copyright 2020 Vector Creations Ltd
```

Comment:
Please use "Copyright 2022 New Vector Ltd" for new added files

Reply:
Fixed in last commit.